nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,61 @@
+ * ***multi-instance***: queues are stored in multiple catalogs on more than one SQL Server instance (Versions 2.1 to 2.x).
+
+NOTE: To properly identify the chosen deployment option all queues that the endpoint interacts with need to be taken into consideration, including error and audit queues. If either of them are stored in a separate SQL Server instance then the deployment is a *multi-instance* one.
+
+The transport will route messages to destination based on the configuration. If no specific configuration has been provided for a particular destination, the transport assumes the destination has the same configuration as the sending endpoint (i.e. identical schema, catalog and instance name). If the destination has a different configuration and it hasn't been provided, then exception will be thrown immediately, because the transport wouldn't be able to connect to the destination queue.
+
+# Modes overview
+
+## Default (single schema, single catalog, single SQL Server instance)
+
+- Supported in Versions 1.0 and higher.
+- Has simple configuration and setup.
+- Doesn't require Distributed Transaction Coordinator (MS DTC).
+- The snapshot (backup) of the entire system state can be done by backing up a single database. It is especially useful if business data is also stored in the same database.
+- Can be monitored with ServiceControl.
+
+## Multi-schema
+
+- Supported in Versions 2.1 and higher.
+- Has simple configuration and setup.
+- Doesn't require Distributed Transaction Coordinator (MS DTC).
+- The snapshot (backup) of the entire system state is done by backing up a single database. It is especially useful if business data is also stored in the same database.
+- Enables security configuration on a schema level.
+- Can't be monitored with ServiceControl.
+
+## Multi-instance (and multi-catalog)
+
+- Supported in Versions 2.1 to 2.x, legacy support is offered in Versions 3.x, won't be supported directly in Versions 4.0 and higher.
+- Requires Distributed Transaction Coordinator, or using Outbox and storing business data in the same database as Outbox data.
+- Can't be monitored with ServiceControl.
+
+## Multi-instance with store-and-forward
+
+SQL Server transport does not support store-and-forward mechanism natively. It means that in *multi-instance* mode if a remote endpoint's infrastructure (e.g. DTC or SQL Server instance) is down, then messages can't be delivered. That makes this particular endpoint, and all endpoints depending on it, unavailable. 
+
+The problem can be addressed using [Outbox](/nservicebus/outbox/) feature. In addition Outbox can be used to avoid escalating transactions to DTC, given that each endpoint has its own database in which it stores input queue, error and audit queues, and the user data. That means it's not possible to avoid distributed transactions in case when any of the queues (including error and audit) is on a different SQL Server instance or catalog. 
```

Comment:
This shouldn't start as a new paragraph. I think this should be a continuation of the previous paragraph, so it reads more continuously. 

>The problem can be addressed using [Outbox](/nservicebus/outbox/) feature.

should be 
This problem can be addressed by using the [Outbox](/nservicebus/outbox/) feature.

>In addition Outbox can be used to avoid escalating transactions to DTC, given that each endpoint has its own database in which it stores input queue, error and audit queues, and the user data. That means it's not possible to avoid distributed transactions in case when any of the queues (including error and audit) is on a different SQL Server instance or catalog. 

The difference between when its possible to avoid DTC using Outbox and when it's not isn't very clear. How about the below? Did I understand this correctly? 

Also, the Outbox feature can be used to avoid escalating transactions to DTC, when each endpoint has a separate database for storing queues and business data on the same SQL Server instance. However, it's not possible to avoid distributed transactions when any of the queues (including error and audit) are on a different SQL Server instance or catalog.